includes/polling/sensors/state/fortigate.inc.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * fortigate.inc.php
+ *
+ * LibreNMS state sensor state polling module for Fortigate Firewalls
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @link       http://librenms.org
+ * @copyright  2020 Net Entertainment AB
+ * @author     Patrik Jonsson <patrik.jonsson@gmail.com>
+ */
+if ($device['os'] == 'fortigate') {
+    if (in_array($sensor['sensor_type'], ['clusterState', 'haSyncStatus'])) {
+        // Fetch the cluster members and their data
+        $fgHaStatsEntryOid = '.1.3.6.1.4.1.12356.101.13.2.1.1';
+        $haStatsEntries = snmpwalk_cache_multi_oid($device, $fgHaStatsEntryOid, [], 'FORTINET-FORTIGATE-MIB');
```

Comment:
You are using the MIB here, you might as well use the textual oid instead of the numeric oid.

Reply:
I tried that first but got `false` as a result of the walk. Will try again.

Reply:
try `fgHaTables`